CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# 93.5.0
+
+This minor release introduces new APIs for Slider. The track height and the
+visibility of track tick marks can be customized.
+
+## New features
+
+`MDCSlider` allows customization of the track height.
+
+**Objective-C**
+
+```objc
+MDCSlider *slider = [[MDCSlider alloc] init];
+slider.trackHeight = 4;
+```
+
+**Swift**
+
+```swift
+let slider = MDCSlider()
+slider.trackHeight = 4
+```
+
+`MDCSlider` also allows controlling the display of the track tick marks for
```

Comment:
```
`MDCSlider` now also allows
```

Reply:
Thanks, Jeff. We spoke offline and agreed that correcting this wasn't worth the trade-off in time for presubmits given the urgent deadline for this Hotfix.